reconcile/utils/saasherder.py

```diff
@@ -441,12 +441,11 @@ def _process_template(self, options):
                 try:
```

Comment:
The comment on L437 could use some help. (Latent; I should have fixed it when I factored out _parameter_value_needed.)

Reply:
what would you change it to? (also, it's Sunday).

Reply:
Probably just:

```python
            # This relies on IMAGE_TAG already being calculated.
```

Reply:
done